packages/did-core-test-server/suites/did-resolution/did-resolution.js

```diff
@@ -1,57 +1,44 @@
-const expectAsciiString = require('./utils').expectAsciiString;
-const expectXmlDateTimeNormalizedToUtcWithoutPrecision = require('./utils').expectXmlDateTimeNormalizedToUtcWithoutPrecision;
-const expectMediaType = require('./utils').expectMediaType;
-const expectConformantDid = require('./utils').expectConformantDid;
-const expectConformantDidDocument = require('./utils').expectConformantDidDocument;
-const expectConformantMetadataStructure = require('./utils').expectConformantMetadataStructure;
-const expectKnownConformantMediaType = require('./utils').expectKnownConformantMediaType;
-const expectConformantRepresentation = require('./utils').expectConformantRepresentation;
+const utils = require('../resolution-utils');
 
-const didResolutionTests = (execution, expectedOutcome) => {
+const didResolutionTests = (execution, expectedOutcome, implementation) => {
   const { did, resolutionOptions } = execution.input;
   const { didResolutionMetadata, didDocument, didDocumentStream, didDocumentMetadata } = execution.output;
   describe(did + ' (expected outcome: ' + expectedOutcome + ')', () => {
-    it.todo('All conformant DID resolvers MUST implement the DID resolution functions for at least one DID method and MUST be able to return a DID document in at least one conformant representation.');
     describe('did', () => {
       it('This input is REQUIRED and the value MUST be a conformant DID as defined in § 3.1 DID Syntax.', async () => {
         expect(did).not.toBeFalsy();
         if (! didResolutionMetadata.hasOwnProperty('error') || didResolutionMetadata['error'] !== 'invalidDid') {
-          expectConformantDid(did);
+          expect(did).toBeValidDid();
         }
       });
     });
     describe('resolutionOptions', () => {
       it('A metadata structure.', async () => {
-        expectConformantMetadataStructure(resolutionOptions);
+        utils.expectConformantMetadataStructure(resolutionOptions);
       });
       it('This input is REQUIRED, but the structure MAY be empty.', async () => {
         expect(resolutionOptions).not.toBeFalsy();
       });
     });
     describe('didResolutionMetadata', () => {
       it('A metadata structure.', async () => {
-        expectConformantMetadataStructure(didResolutionMetadata);
+        utils.expectConformantMetadataStructure(didResolutionMetadata);
       });
       it('This structure is REQUIRED, and in the case of an error in the resolution process, this MUST NOT be empty.', async () => {
         expect(didResolutionMetadata).not.toBeFalsy();
-        if (isErrorExpectedOutcome(expectedOutcome)) {
+        if (utils.isErrorExpectedOutcome(expectedOutcome)) {
           expect(Object.keys(didResolutionMetadata)).not.toHaveLength(0);
         }
       });
       it('If resolveRepresentation was called, this structure MUST contain a contentType property containing the Media Type of the representation found in the didDocumentStream.', async () => {
```

Comment:
This goes for all `it()` statements in the test suite. I'm afraid that if we don't prefix each test with the section number, that it's going to be even more difficult for the person doing the test reporting logic to match each test appropriately and automatically tally the "multiple independent implementations" of each feature.